notebooks/speechbrain-emotion-recognition/speechbrain-emotion-recognition.ipynb

```diff
@@ -0,0 +1,454 @@
+{
```

Comment:
<p>Could you please add more information about output parameters ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2064/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done